packages/broker/src/plugins/operator/MaintainTopologyService.ts

```diff
@@ -16,72 +13,49 @@ function toStreamIDSafe(input: string): StreamID | undefined {
     }
 }
 
-function singletonSet<T>(element: T): Set<T> {
-    return new Set<T>([element])
-}
-
 export class MaintainTopologyService {
     private readonly streamrClient: StreamrClient
-    private readonly operatorClient: OperatorClient
+    private readonly maintainTopologyHelper: MaintainTopologyHelper
     private readonly subscriptions = new Multimap<StreamID, Subscription>()
     private readonly synchronizer = new SetMembershipSynchronizer<StreamID>()
     private readonly concurrencyLimit = pLimit(1)
+    private readonly logger: Logger
 
-    constructor(streamrClient: StreamrClient, operatorClient: OperatorClient) {
+    constructor(streamrClient: StreamrClient, maintainTopologyHelper: MaintainTopologyHelper, logger: Logger) {
         this.streamrClient = streamrClient
-        this.operatorClient = operatorClient
+        this.maintainTopologyHelper = maintainTopologyHelper
+        this.logger = logger
     }
 
     async start(): Promise<void> {
-        this.operatorClient.on('addStakedStream', this.onAddStakedStream)
-        this.operatorClient.on('removeStakedStream', this.onRemoveStakedStream)
-        await this.operatorClient.start()
-        const rawStreamIds = await this.operatorClient.getStakedStreams()
-        const blockNumber = 10 // TODO: fix properly
-        const streamIds = new Set(compact([...rawStreamIds].map(toStreamIDSafe)))
-        const { added } = this.synchronizer.ingestSnapshot(streamIds, blockNumber)
-        for (const streamId of added) {
-            await this.addStream(streamId, blockNumber)
-        }
-        logger.info('Started')
+        this.logger.info('Starting MaintainTopologyService')
```

Comment:
```suggestion
        this.logger.info('Starting')
```

That way the log message (using the static logger) would become:
```
INFO [2023-06-19T07:20:06.010] (MaintainTopologyService  ): Starting
```
(No need to re-state the class name)